ha/virsh/vm_utils.sh

```diff
@@ -167,7 +167,7 @@ backoff() {
   BACKOFF_FACTOR="${2:-2}"
   MAX_RETRIES="${3:-7}"
   for i in $(seq 0 ${MAX_RETRIES}); do
-    if [ i -gt 0 ]; then
+    if [ $i -gt 0 ]; then
```

Comment:
ugh, I missed that. We could have gone with `((i > 0))` given that this is bash, but good old `[` is also good here.